metadata.sh

```diff
@@ -19,7 +19,7 @@ export OPERATOR_NAME="cassandra"
 
 # More details about KUDO Versioning:
 # https://github.com/kudobuilder/kudo/pull/1028
-export OPERATOR_VERSION="0.1.2"
+export OPERATOR_VERSION="0.2.0"
```

Comment:
let's bump this to `1.0.0` 
cc: @ANeumann82

Reply:
We should bump before the actual release. This is just for the preliminary one that I'm doing to test the release tooling.